plugin/group_replication/src/certifier.cc

```diff
@@ -1223,8 +1247,16 @@ void Certifier::garbage_collect() {
   }
 #endif
 
-  mysql_mutex_unlock(&LOCK_certification_info);
+    mysql_mutex_unlock(&LOCK_certification_info);
+    if (it == certification_info.end()) {
```

Comment:
I don't think we should access this variable without holding the LOCK_certification_info mutex.